extraction/embedded_tweet_extractor.py

```diff
@@ -0,0 +1,22 @@
+"""This script checks whether DOM has embedded tweet tag or not and
+creates and returns the ETweet object"""
+
+from extraction.data_models.embedded_tweet import ETweet
+from extraction.interface_content_extractor import IContentExtractor
+
+
+class ETweetExtractor(IContentExtractor):
+    """This class inherits IContentExtractor for extracting embedded tweets"""
+
+    def validate_and_extract(self, node):
```

Comment:
can you add return type here.

Reply:
None won't be accepted as return value if I add ETweet Class as return type.